blocks/library/button/index.js

```diff
@@ -182,13 +170,13 @@ export const settings = {
 
 	attributes: blockAttributes,
 
-	getEditWrapperProps( attributes ) {
-		const { align, clear } = attributes;
-		const props = {};
+	supports: {
+		align: true,
+		alignWide: false,
+	},
 
-		if ( 'left' === align || 'right' === align || 'center' === align ) {
-			props[ 'data-align' ] = align;
-		}
+	getEditWrapperProps( { clear } ) {
+		const props = {};
```

Comment:
Since `getEditWrapperProps` can return `undefined`, this could potentially be simplified to:

```js
getEditWrapperProps( { clear } ) {
	if ( clear ) {
		return { 'data-clear': 'true' };
	}
}
```